userguide/jails.rst

```diff
@@ -1057,7 +1022,7 @@ Jail Software
 
 A jail is created with no software aside from the core packages
 installed as part of the selected version of FreeBSD. Software in a
-jail is managed by navigating to the :guilabel:`Shell` and logging
+jail is managed by going to the :guilabel:`Shell` and logging
 into the jail with :command:`iocage console`. In this example,
```

Comment:
But "iocage console" does not log in to the jail, it requires a jailname.  Use the :samp: command from above.